docs/guides/es2k-guide.md

```diff
@@ -66,15 +68,16 @@ appropriate path instead of ./install.
 we are creating directories and copying config file at system level.
 
 ```bash
-sudo ./scripts/es2k/copy_config_files.sh $IPDK_RECIPE $SDE_INSTALL
+source $P4CP_INSTALL/sbin/setup_env.sh $P4CP_INSTALL $SDE_INSTALL $DEPEND_INSTALL
+sudo bash $P4CP_INSTALL/sbin/copy_config_files.sh $P4CP_INSTALL $SDE_INSTALL
```

Comment:
Should remove the `bash` command here, similar to the dpdk comment that Derek made yesterday